src/api/environment.cc

```diff
@@ -556,6 +556,15 @@ Maybe<bool> InitializeContextRuntime(Local<Context> context) {
   Isolate* isolate = context->GetIsolate();
   HandleScope handle_scope(isolate);
 
+  // Delegate the code generation validation to
+  // node::ModifyCodeGenerationFromStrings.
+  bool is_code_generation_from_strings_allowed =
+      context->IsCodeGenerationFromStringsAllowed();
+  context->AllowCodeGenerationFromStrings(false);
```

Comment:
Can you add some comments here explaining why this is set to false? I had to read https://github.com/nodejs/node/pull/43428/files#r914033892 to figure out why we couldn't simply let it be the default value. Similarly it would be useful to explain that the value from `context->IsCodeGenerationFromStringsAllowed()` is refreshed by V8 according to the runtime flags, we are just propagating that refreshed value to the embedder data here.

Reply:
Thanks for the suggestion! Updated.